test/parallel/test-http2-goaway-delayed-request.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
```

Comment:
I haven't tested it, but it doesn't look like this test uses any crypto related API (or am I missing something?)
```suggestion
```

Reply:
You need crypto to use http2.